cfpb/cfgov-refresh
# Ask CFPB: Spanish page improvements

Add pagination, disclaimer, and mobile link style updates to Spanish Ask pages

## Additions

- New Spanish disclaimer language on Spanish answer pages
- Mobile style for related question links on Spanish answer pages

## Changes

- Hide pagination on Spanish Ask pages when there's only one page of results

## Testing

1. Check out branch and run gulp styles
2. Navigate to [Spanish tag with only one page of results](http://localhost:8000/es/obtener-respuestas/buscar-por-etiqueta/dealer/) and verify that pagination does not appear
3. Navigate to [Spanish answer page](http://localhost:8000/es/obtener-respuestas/que-efecto-tendria-sobre-mi-credito-comparar-diferentes-ofertas-de-prestamos-para-vehiculos-es-763/) and verify new disclaimer language in `Acerca de estas respuestas` section and mobile style for `Preguntas similares`

## Screenshots

### Preguntas similares
#### Before
<img width="377" alt="screen shot 2017-06-20 at 10 12 08 am" src="https://user-images.githubusercontent.com/778171/27401177-62c22bf8-5677-11e7-8aaf-bda6501b53dd.png">  

  
#### After
<img width="412" alt="screen shot 2017-06-20 at 10 11 40 am" src="https://user-images.githubusercontent.com/778171/27401200-6e5b8cb6-5677-11e7-9765-1d61dbcdeb1f.png">

### Pagination
#### Before
<img width="966" alt="screen shot 2017-06-20 at 10 02 25 am" src="https://user-images.githubusercontent.com/778171/27401248-880e0f9e-5677-11e7-9132-29c88ef0050f.png">  
  
#### After
<img width="959" alt="screen shot 2017-06-20 at 10 02 05 am" src="https://user-images.githubusercontent.com/778171/27401258-8e6ff15e-5677-11e7-8ea2-0b0a28be74e5.png">

### Disclaimer
#### Before
<img width="310" alt="screen shot 2017-06-20 at 9 56 11 am" src="https://user-images.githubusercontent.com/778171/27401268-98c3ae48-5677-11e7-9f82-184273528bd9.png">  

#### After
<img width="296" alt="screen shot 2017-06-20 at 9 55 54 am" src="https://user-images.githubusercontent.com/778171/27401280-9fbbfd18-5677-11e7-8ef0-093bf77d6e20.png">



## Checklist
